modules/lightning_features/lightning_layout/modules/lightning_landing_page/config/install/core.entity_view_display.node.landing_page.full.yml

```diff
@@ -49,7 +49,7 @@ third_party_settings:
         uuid: 5af47416-fa98-4dfb-9761-fc497a47a76f
         label: 'Three Column'
         weight: 0
-        layout: threecol_25_50_25_stacked
+        layout: layout_threecol_25_50_25_stacked
```

Comment:
There is the change in the layout config name. We require an hook_post_update_name for the sites which are using this layout.